src/modelbench/run.py

```diff
@@ -269,23 +269,25 @@ def calibrate(update: bool, file) -> None:
 
 def update_standards_to(standards_file):
     reference_suts = [
-        ModelGaugeSut.for_key("mistral-7b"),
+        ModelGaugeSut.for_key("gemma-2-9b-it-hf"),
+        ModelGaugeSut.for_key("llama-3.1-8b-instruct-turbo-together"),
     ]
     if not ensure_ensemble_annotators_loaded():
         print("Can't load private annotators needed for calibration")
         exit(1)
 
-    benchmarks = [GeneralPurposeAiChatBenchmark()]
-    for l in Locale:
+    benchmarks = []
+    for l in [Locale.EN_US]:
```

Comment:
I might have missed this, but is 1.0 only going to include english prompts?

Reply:
Yup. @bollacker knows more, but there were a number of challenges, so new languages will come shortly after 1.0.

Reply:
Ah ok, sounds good